duckduckgo/duckduckgo-privacy-extension
# Multi-step Onboarding: Navigate back to DuckDuckGo post-install

**Reviewer:**
@jdorweiler 

## Description:
- Removes most of #435 
- Adds logic to detect URL param when installing, and take the user back to DDG instead of opening the post-install page

## Steps to test this PR:
- Visit https://$dev.duckduckgo.com?t=hx
- Click "Add DuckDuckGo to Chrome"
    - Url should update to `https://$dev.duckduckgo.com?t=hx&natb=v###-#rx&cp=atbhc&npi=1`
    - Chrome store should open
- Open the Chrome Extensions tab and install the dev extension build
- Browser should focus on the DDG Tab, and modify the URL, and page should load revealing step 2 of the multi-step onboarding.

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
